go-controller/pkg/ovn/gateway_init.go

```diff
@@ -140,6 +141,16 @@ func gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet, hostSubnets []*n
 			kapi.ProtocolUDP:  k8sNSLbUDP,
 			kapi.ProtocolSCTP: k8sNSLbSCTP,
 		}
+		workerK8sNSLbTCP, workerK8sNSLbUDP, workerK8sNSLbSCTP, err := loadbalancer.GetWorkerLoadBalancers(nodeName)
```

Comment:
can you add the gr prefix to protoLBMap / k8sNSLbTCP to make it clear the difference with worker*?

Reply:
OK will change the protoLBMap var name to include GR.